core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -35,6 +35,7 @@
 import static com.google.common.base.Preconditions.checkNotNull;
 import static com.google.common.base.Preconditions.checkState;
 
+import com.google.census.CensusContextFactory;
```

Comment:
> I would much prefer an approach that didn't add a dependency but instead allowed me to hook into Atlas(the Netflix metrics library).

@jhspaybar Would you mind opening an issue at https://github.com/google/census-java to describe your use case and the type of API you would need for recording stats?